Sources/SmileID/Classes/SelfieCapture/SelfieViewModel.swift

```diff
@@ -76,55 +77,55 @@ public class SelfieViewModel: ObservableObject, ARKitSmileDelegate {
         self.skipApiSubmission = skipApiSubmission
         self.extraPartnerParams = extraPartnerParams
         self.localMetadata = localMetadata
-        
+
         cameraManager.$status
             .receive(on: DispatchQueue.main)
             .filter { $0 == .unauthorized }
             .map { _ in AlertState.cameraUnauthorized }
             .sink { alert in self.unauthorizedAlert = alert }
             .store(in: &subscribers)
-        
+
         cameraManager.sampleBufferPublisher
             .merge(with: arKitFramePublisher)
             .throttle(for: 0.35, scheduler: DispatchQueue.global(qos: .userInitiated), latest: true)
-        // Drop the first ~2 seconds to allow the user to settle in
+            // Drop the first ~2 seconds to allow the user to settle in
```

Comment:
Comment says first two seconds, but the code in the next line reflects 5.. is this discrepancy intentional?

Reply:
Got it.. 
LGTM